src/css/base.scss

```diff
@@ -35,6 +36,48 @@ main {
   margin-left: 296px;
 }
 
+li {
+ display: block;
+ transition-duration: 0.5s;
+}
+
+li:hover {
+  cursor: pointer;
+}
+
+ul li ul {
+  visibility: hidden;
+  opacity: 0;
+  position: absolute;
+  transition: all 0.5s ease;
+  margin-top: 1rem;
+  left: 0;
+  display: none;
+}
+
+ul li:hover > ul,
+ul li ul:hover,
+ul li ul:focus {
+  visibility: visible;
+  opacity: 1;
+  display: block;
+}
+
+ul li ul li {
```

Comment:
I know you mentioned you're aiming to refactor to BEM which is a great move! This sort of structure can definitely create some tricky specificity bugs or just be tough to read and work with when extending upon it or changing styles.